netlify.toml

```diff
@@ -1,3 +1,3 @@
 [build]
   functions = "netlify-lambda-lib"
-  command = "npm run build"
+  command = "npm run download && npm run build"
```

Comment:
Can you please move Netlify fix into separate PR?

Reply:
addesssed https://github.com/graphql/swapi-graphql/pull/187